src/frontend/planner_test/tests/testdata/output/index_selection.yaml

```diff
@@ -740,3 +740,20 @@
   batch_plan: |-
     BatchExchange { order: [], dist: Single }
     └─BatchScan { table: people_names, columns: [people_names.first_name, people_names.last_name, people_names.info], scan_ranges: [people_names.CONCAT_OP = Utf8("John Smith")], distribution: SomeShard }
+- name: funtional index const eval
+  sql: |
+    create table a(v int);
+    create index b on a((1+1)*v);
```

Comment:
Is it possible to check the plan of `create index` in the planner test case as well?